solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,151 @@
+"""Simulating an ATM. """
+
+import sys
+
+ATM_FILE = 1  # The number 1 refers to the place of the file as a parameter
+
+
+def check_the_balance(costumer_id, costumers_dictionary):
+    """Check the state of costumer's money in the bank.
+
+    Args:
+    costumer_id (string): costumer id.
+    costumers_dictionary (dictionary): dictionary of all costumers in
+                                       the bank.
+    """
+    balance = int(costumers_dictionary[costumer_id][1])
+    return balance
+
+
+def cash_withdrawal(costumer_id, amount_to_withdrawal, costumers_dictionary):
+    """Withdrawal from the ATM amount of money.
+
+    Args:
+    costumer_id (string): costumer id.
+    amount_to_withdrawal (integer): amount the costumer wants to withdrawal.
+    costumers_dictionary (dictionary): dictionary of all costumers in
+                                       the bank.
+    """
+    balance_now = check_the_balance(costumer_id, costumers_dictionary)
+    if amount_to_withdrawal > 0:
+        if balance_now >= amount_to_withdrawal:
+            new_balance = balance_now - amount_to_withdrawal
+            costumers_dictionary[costumer_id][1] = str(new_balance)
+            return True
+    return False
+
+
+def cash_deposit(costumer_id, amount_to_deposit, costumers_dictionary):
+    """deposit to the bank some of money.
+
+    Args:
+    costumer_id (string): costumer id.
+    amount_to_deposit (integer): amount the costumer wants to deposit.
+    costumers_dictionary (dictionary): dictionary of all costumers in
+                                       the bank.
+    """
+    if amount_to_deposit > 0:
+        balance_now = check_the_balance(costumer_id, costumers_dictionary)
+        new_balance = balance_now + amount_to_deposit
+        costumers_dictionary[costumer_id][1] = str(new_balance)
+        return True
+    return False
+
+
+def change_password(costumer_id, new_password, costumers_dictionary):
+    """Changing the costumer's ATM password.
+
+    Args:
+    costumer_id (string): costumer id.
+    new_password (string): the new password the costumer wants to change to.
+    costumers_dictionary (dictionary): dictionary of all costumers in
+                                       the bank.
+    """
+    if len(new_password) == 4 and new_password.isdigit():
+        costumers_dictionary[costumer_id][0] = new_password
+        return True
+    return False
+
+
+def main():
+    costumers_dictionary = {}
+    try:
+        with open(sys.argv[ATM_FILE], 'r') as atm_file:
+            for line in atm_file:
+                costumers_dictionary[line.split()[0]] = line.split()[1:]
+
+    except FileNotFoundError:
+        print("Your file path is incorrect. Please check it.")
+
+    try:
+        print("welcome to bank!\n")
+        costumer_id = input("enter your id:\n")
+        while costumer_id != str(-1):
```

Comment:
Anyone can do any action he wants with  only knowing the id? its very easy to steal in this atm